ringpop.go

```diff
@@ -60,6 +60,11 @@ type Interface interface {
 
 	HandleOrForward(key string, request []byte, response *[]byte, service, endpoint string, format tchannel.Format, opts *forward.Options) (bool, error)
 	Forward(dest string, keys []string, request []byte, service, endpoint string, format tchannel.Format, opts *forward.Options) ([]byte, error)
+
+	// extend the functionality of self eviction to ringpop
+	// Mockery didn't allow me to embed the interface so we make things WET here
```

Comment:
Haha, was a joke to DRY (Don't Repeat Yourself) when something is not dry it is wet, but I will remove the joke from the comments.